_search-plugins/sql/sql/functions.md

```diff
@@ -225,3 +225,63 @@ The results contain documents that match the wildcard expression:
 | account_number | address
 :--- | :---
 1 | 880 Holmes Lane
+
+## Nested query
+
+The nested function is used in the `SELECT` clause to unnest nested object type collections. The nested collection is flattened and a cartesian product is returned when querying against two nested objects.
+
+### Syntax
+
+The `field_expression` parameter is required, and the `path_expression` parameter is optional. Dot notation is used to show nesting level for `field_expression` and `path_expression` parameters. For example `nestedObj.innerFieldName` denotes a field nested one level. If the user does not provide the `path_expression` parameter it will be generated dynamically. For example the field `user.office.cubicle` would dynamically generate the path `user.office`.
+
+```sql
+nested(field_expression | field_expression, path_expression)
```

Comment:
Would it be worth putting the example before the description so you can see the layout of the expression before the explanation?

Reply:
Doesn't tend to be the standard way that the examples are presented on the documentation. I'd like to conform with the other pages.